single-program/.env.example

```diff
@@ -0,0 +1,8 @@
+CONTRACT_ID = 
+NEAR_ENV = 
+NODE_ENV = 
+ACCOUNT_ID = 
+METHOD_NAME = 
+ATTACHED_DEPOSIT = 
+PRIVATE_KEY = 
+MAX_GAS = "300000000000000"
```

Comment:
Why `MAX_GAS` is required in their project env?

Reply:
Last I checked it is needed in the x2earn sdk

Reply:
If it is required in x2earn sdk, this variable should be directly in the contract code it self but here. Please check with vaulstein

Reply:
ok